codeMetadata.go

```diff
@@ -23,6 +25,7 @@ func CodeMetadataFromBytes(bytes []byte) CodeMetadata {
 
 	return CodeMetadata{
 		Upgradeable: (bytes[0] & METADATA_UPGRADEABLE) != 0,
```

Comment:
Extract the byte indexes as constants too, e.g.
UPGRADEABLE_BYTE = 0
UPGRADEABLE_BIT_MASK = 1
etc.
I think it is easier to read

Reply:
Also Go doesn't like ALL_CAPS, so maybe rename to
upgradeableByte = 0,
upgradeableBitMask = 1

Reply:
would leave as it is.